### ISSUE TITLE
Unclosed Requests and the Diamond Collectors

### ISSUE THREAD

**Issue by LunarWatcher** [SO-Close-Vote-Reviewers/UserScripts#149]
TL;DR: The Unclosed Request Review userscript seems to duplicate moderator diamonds. 

With the userscript enabled:

![](https://i.imgur.com/9vDe90W.png)

When it's disabled:

![image](https://user-images.githubusercontent.com/29489988/53025102-9e5e5d80-3460-11e9-81fd-5db57dfba9b4.png)

Additional screenshots when the userscript is enabled:

![image](https://user-images.githubusercontent.com/29489988/53025136-a918f280-3460-11e9-92c6-9c24e6db7561.png)
![image](https://user-images.githubusercontent.com/29489988/53025155-b209c400-3460-11e9-9822-73a6cedd56e0.png)

All of which go back to single diamonds when the userscript is disabled. Reproduced in SOBotics and SOCVR. Also reproduced [by someone else](https://chat.stackoverflow.com/transcript/message/45396718#45396718).

**Comment by makyen** [comment]
Where *exactly* are you seeing these? Is it in the user-info popup (i.e. the popup that opens when you click on a moderator's username in chat)?

What version number of the beta version of the URRS are you using? The [current beta version is 1.1.3.1](https://github.com/SO-Close-Vote-Reviewers/UserScripts/blob/URRS-Mak-new-version/UnclosedRequestReview.user.js). An issue along these lines was addressed in 1.1.3.0. Are you, perhaps, using an older version? If so, what userscript manager are you using? I ask, because there should have been an automatic update and I'd like to explore why there wasn't.

**Comment by LunarWatcher** [comment]
I appear to have used the SOBotics version and not the SOCVR version. I thought there only was one version ^^" 

[status-no-repro] in 1.1.3.1, which means I posted in the wrong repo.

**Comment by makyen** [comment]
np. This was the first that I was aware of another version, or that SOBotics had additional features that were desired (other than getting status for comments, which is a bit more complex). It looks like we'll get a PR from Filnor to merge the changes they made for SOBotics in here. I view that as quite beneficial, so I'm glad you reported it here.

**Comment by double-beep** [comment]
@makyen in the SOCVR version I see the diamond before the moderator name. Is that right?

**Comment by makyen** [comment]
@double-beep If you mean before the name when viewing a main chat page, then, yes, that is something the script adds. If you don't like it, it can be disabled from the options page.
